### ISSUE TITLE
Enhancement request: indicate sites that use browser whitelisting

### ISSUE THREAD

**Issue by fredemmott** [Nitrokey/dongleauth#243]
For example, Google [claims](https://security.googleblog.com/2019/08/making-authentication-even-easier-with_12.html) that Google accounts support WebAuthn, however, attempting to use it in any browser other than Chrome results in this:

![Screen Shot 2019-12-18 at 10 16 39 AM](https://user-images.githubusercontent.com/360927/71112983-8723f580-2181-11ea-870f-fa1e22289d55.png)

This makes WebAuthn support seem more like a chrome advertisement than a security feature.

**Comment by seonwoolee** [comment]
I second this. Vanguard falls into this category as well

**Comment by fredemmott** [comment]
Yeah, I'm hopeful about Vanguard: this month they required U2F users to re-enroll, which I'm hoping is a move to webauthn and then broader support.

**Comment by jscott0** [comment]
Duo also whitelists. I need to choose 'Desktop site' from Chrome on Android for it to allow me to use my key.
